app/assets/stylesheets/omni_nav/omni_nav_v2/off-canvas-nav.scss

```diff
@@ -84,10 +84,70 @@
         div.toggle-logo {
           float: left;
           width: 50%;
-
           padding: 18px;
           position: relative;
 
+          .asbe {
```

Comment:
All these school/college styles are copied directly from the current version. They were referencing version 1 stylesheets so I had to copy them here to make sure the two omninavs are independent.

Reply:
👍